pycti/utils/opencti_stix2.py

```diff
@@ -1507,7 +1507,7 @@ def generate_export(self, entity: Dict, no_custom_attributes: bool = False) -> D
             and "valid_until" in entity
             and entity["valid_from"] == entity["valid_until"]
         ):
-            del entity["valid_from"]
+            entity["valid_until"] = entity["valid_from"] + datetime.timedelta(seconds=1)
```

Comment:
Hello ! 
Warning : you will get an error because you are concatenating a "string" with a "datetime".

Proposal, something like that?
```python
  valid_until_converted_datetime = datetime.strptime(entity["valid_until"], '%Y-%m-%dT%H:%M:%S.%fZ')
  new_valid_until = valid_until_converted_datetime + timedelta(seconds=1)
  valid_until_converted_string = new_valid_until.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
  entity["valid_until"] = valid_until_converted_string
```